CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - feat: add `debug` exporter [#1268]
 - feat: add `routing` connector [#1268]
+- feat(ci): add fips build & msi for windows/amd64 [#1269]
```

Comment:
If we're only providing dev builds for now, let's skip the changelog entry to avoid confusing users.